features/i8n.feature

```diff
@@ -0,0 +1,118 @@
+Feature: Pro supports multiple languages
+
+    @series.all
+    @uses.config.machine_type.lxd.container
+    @uses.config.contract_token
+    Scenario Outline: Pro client's commands run successfully
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        ## Change the locale
+        When I run `locale-gen uk_UA.UTF-8` with sudo
+        And I run `update-locale LANG=uk_UA.UTF-8` with sudo
+        And I reboot the machine
+        And I run `cat /etc/default/locale` as non-root
+        Then stdout matches regexp:
+        """
+        LANG=uk_UA.UTF-8
+        """
+        #Attach invalid token
+        When I verify that running `pro attach INVALID_TOKEN` `with sudo` exits `1`
+        Then stderr matches regexp:
+            """
+            Invalid token. See https://ubuntu.com/pro
+            """
```

Comment:
I think we're standardizing on not indenting these text sections, so this applies to all of these
```suggestion
        """
        Invalid token. See https://ubuntu.com/pro
        """
```